package.json

```diff
@@ -6,7 +6,6 @@
   "devDependencies": {
     "babel-cli": "6.8.0",
     "babel-plugin-lodash": "3.1.0",
-    "babel-plugin-rewire": "1.0.0-rc-2",
```

Comment:
I tried that, but because the `lib` isn't committed to the repository, I have to build the lib manually (hence the travis craziness) :-)